Hooks/ilhooks.md

```diff
@@ -292,23 +294,30 @@ You can emit any kind of opcode, with or without an operand. As a reminder, ther
 [here](https://en.wikipedia.org/wiki/List_of_CIL_instructions). You'll see most of the ones you'll usually use in
 the process of investigating the IL code you're hooking. Covering the functionality is out of scope for this
 documentation, but a few commonly used ones include:
-* ldc.i4
-* ldstr
-* ldarg
-* ldfld
-* callvirt
+* ldc.i4 (load constant 4-byte integer)
+* ldc.r4 (load constant 4-byte float)
+* ldstr (load a string constant)
+* ldloc (load a local variable)
+* ldarg (load a method argument)
+* ldfld (load a field of a class)
+* callvirt (call an instance method of an object)
+
+> Note: many of these instructions have several shorthand variants, such as `ldc.i4.0`. These are functionally
+identical to the normal opcode (such as `ldc.i4 0`), but use a different opcode and no operand. When matching,
```

Comment:
I'm not sure if it's relevant but can we explain that ldarg.0 is this and the other params are ldarg.1 and so on.

Reply:
Idk where this is said so imo it makes sense to say it here

Reply:
I believe that's self-explanatory, and that it's not in scope of this documentation to explain every single IL opcode. I have
linked to a full list of opcodes several times, along with specifying there are "several variants" for each of these